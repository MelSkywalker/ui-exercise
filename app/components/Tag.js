import React from 'react';
import './tag.scss';

const Tag = ({tagName, handleFilterItems}) => {
	return(
		<button className="tag" onClick={handleFilterItems} name={tagName}>
			{tagName}
		</button>
	);
}

export default Tag;
