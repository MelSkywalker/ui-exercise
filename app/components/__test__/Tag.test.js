import React from 'react';
import { shallow } from 'enzyme';

import Tag from '../Tag';

describe('components | Tag', () => {
	it('it renders the component', () => {
		const component = shallow(<Tag />);
		expect(component).toMatchSnapshot();
	});
});
