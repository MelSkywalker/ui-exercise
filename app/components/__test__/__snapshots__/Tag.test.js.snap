// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Tag it renders the component 1`] = `
<button
  className="tag"
  name="work"
  onClick={[MockFunction]}
>
  work
</button>
`;
